index.js

```diff
@@ -1,137 +1,63 @@
-let library = {
-  nextId: 0, // return to 0;
-  books: [],
-  add: (title, author) => {
-    const newBook = {
-      id: library.nextId,
-      title,
-      author,
-    };
+/* eslint-disable max-classes-per-file */
 
-    library.books.push(newBook);
+// Select Elements
+const library = document.querySelector('.library');
+const bookName = document.querySelector('.name');
+const bookAuthor = document.querySelector('.author');
+document.body.style.backgroundColor = 'red';
 
-    library.nextId += 1;
-    return newBook;
-  },
-  remove: (id) => {
-    library.books = library.books.filter((book) => book.id !== id);
-  },
-};
+// Create Empty Array
+const books= [];
```

Comment:
- [ ] Good job so far :+1: You are to use JavaScript classes instead of objects and arrays but you can use objects and arrays but it has to be inside the classes. Kindly remove this `books` array here as it is not used inside a class :+1: 
- Requirements:
![Screenshot from 2022-08-11 07-13-18](https://user-images.githubusercontent.com/85113567/184074648-522c579d-b5a3-4635-9873-07e85c644490.png)